src/browser/disabled_frame_group_tracker.ts

```diff
@@ -92,89 +101,110 @@ function handleApiMove(win: OpenFinWindow, delta: RectangleBase) {
             ? 2
```

Comment:
I know is was not introduced on this change set, but can we make the change types either `const`s or `enum`s?